pkg/antctl/raw/multicluster/deploy/member_cluster.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+type memberClusterOptions struct {
+	namespace     string
+	antreaVersion string
+	filename      string
+}
+
+var memberClusterOpts *memberClusterOptions
+
+var memberClusterExamples = strings.Trim(`
+Create the member cluster CRDs and deploy the "antrea-mc-controller" Deployment in Namespace antrea-mcs
+$ antctl mc deploy member-cluster --antrea-version v1.6.0 -n antrea-mcs
+Create the member cluster CRDs and deploy the "antrea-mc-controller" Deployment using pre-downloaded manifest
+$ antctl mc deploy member-cluster -f antrea-multicluster-member.yml
+
+The following CRDs will be created:
+- CRDs: ClusterClaim, ClusterSet, MemberClusterAnnounce, ResourceExportFilter, ResourceExport, ResourceImport, ResourceImportFilter, ServiceExport, ServiceImport
+`, "\n")
+
+func (o *memberClusterOptions) validateAndComplete() error {
+	if o.filename != "" {
+		return nil
+	}
+	if o.namespace == "" {
+		return fmt.Errorf("the Namespace cannot be empty")
+	}
+	if o.antreaVersion == "" {
+		o.antreaVersion = "latest"
+	}
+
+	return nil
+}
+
+func NewMemberClusterCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "member-cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Short:   "Deploy Antrea Multi-cluster to a member cluster",
+		Long:    "Deploy Antrea Multi-cluster to a member cluster in a Namespace",
+		Example: memberClusterExamples,
+		RunE:    memberClusterRunE,
+	}
+	o := &memberClusterOptions{}
+	memberClusterOpts = o
```

Comment:
ditto

Reply:
Sorry. The suggestion above is about deploy from file. This line is not about deploy. So I'm confused.